library/src/main/java/com/pokegoapi/api/listener/SettingsListener.java

```diff
@@ -0,0 +1,19 @@
+package com.pokegoapi.api.listener;
+
+import com.pokegoapi.api.PokemonGo;
+
+/**
+ * Created by RebliNk17 on 09/04/2017.
+ */
+public interface SettingsListener extends Listener {
+
+	/**
```

Comment:
[ant:checkstyle] [WARN] /home/jasper/Documents/PokemonGo/PokeGOAPI-Java/library/src/main/java/com/pokegoapi/api/listener/SettingsListener.java:10: Block comment has incorrect indentation level 1, expected is 2, indentation should be the same level as line 17. [CommentsIndentation]